sdk/hdinsight/arm-hdinsightcontainers/package.json

```diff
@@ -40,6 +40,7 @@
     "@azure-tools/test-credential": "^1.1.0",
     "mocha": "^10.0.0",
     "@types/mocha": "^10.0.0",
+    "tsx": "^4.7.1",
```

Comment:
why this change?

Reply:
https://github.com/Azure/autorest.typescript/commit/30e034376dbdef567f3d870a5e43e590e7468927